packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -320,6 +330,18 @@ function VM-Install-Shortcut{
         Install-ChocolateyShortcut @shortcutArgs
     }
     VM-Assert-Path $shortcut
+
+    # If the file targets a .bat, we should set the shortcut to match (requires shortcut to have been created already)
```

Comment:
if the shortcut is not created, we do not reach this code, or am I missing something?

Reply:
That would actually be exactly what we want! If there is no shortcut, then we don't want to attempt to modify the shortcut icon if it isn't there. :)

Reply:
I meant I found the comment difficult to understand and that could be simplified. I should have marked as nitpick